code/datums/components/thermite.dm

```diff
@@ -3,21 +3,21 @@
 	var/amount
 	var/overlay
 
-	var/static/list/blacklist = typecacheof(
+	var/static/list/blacklist = typecacheof(list(
 		/turf/open/lava,
 		/turf/open/space,
 		/turf/open/water,
-		/turf/open/chasm,
+		/turf/open/chasm)
 		)
```

Comment:
I'm surprised this wasn't found out earlier.

Leave the formatting as it was before, it was like that to prevent as many line differences for things later which add to the list. The extra comma at the end isn't harmful